listenbrainz/webserver/static/js/jsx/follow-users.jsx

```diff
@@ -6,7 +6,10 @@ export class FollowUsers extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      users: props.followList || []
+      users: props.followList || [],
+      saveUrl: props.saveUrl || "https://listenbrainz.org/follow/save",
```

Comment:
For the default value, I would suggest getting the current origin, so that there should be nothing to change if you are running locally:

` `${window.location.origin}/follow/save` `

Reply:
Right, makes sense.